app.py

```diff
@@ -44,3 +51,19 @@ def serve_angular(path):
     ])
     return flask.redirect(target)
   return flask.send_file('dist/client/index.html')
+
+@app.route('/list', methods=['GET'])
+def read():
+    """
+        read() : Fetches documents from Firestore collection as JSON.
+        user_email : Return document that matches query email.
+        all_todos : Return all documents.
```

Comment:
I don't see us supporting (or needing) this function yet - can we remove from docstring?

Reply:
Yes I can remove that